Formula/o/orbiton.rb

```diff
@@ -0,0 +1,25 @@
+class Orbiton < Formula
+  desc "Fast and configuration-free text editor and IDE limited by VT100"
+  homepage "https://orbiton.zip/"
+  url "https://github.com/xyproto/orbiton/archive/refs/tags/v2.64.3.tar.gz"
+  sha256 "f3ebdcdc7cb3502705ceda208510da1a43e65b5e0cb33e57b86564667b48303f"
+  license "BSD-3-Clause"
+  head "https://github.com/xyproto/orbiton.git", branch: "main"
+
+  depends_on "go" => :build
+
+  on_linux do
+    depends_on "xclip" => :test
```

Comment:
That doesn't seem likely since the test only runs the program

Reply:
`o` needs `xsel` or `xclip` on Linux (under X) to copy text to the clipboard or paste text from the clipboard. And `wl-clipboard` is required if running under Wayland. If they are not available, this is the error message:

`error: No clipboard utilities available. Please install xsel, xclip, wl-clipboard or Termux:API add-on for termux-clipboard-get/set.`

However, if `xclip` is installed but X is not running (like when running these GH Actions), `o` just returns with exit status 1.

Running the test with `xvfb-run` when on Linux should fix the issue (testing now).

Reply:
So that means it's a dependency on runtime too, not just `:test`

Reply:
Ah, right! Yes, it is an optional dependency for Linux. Some users prefer Wayland.

Reply:
Optional dependencies does not appear to be allowed in Homebrew Core, so maybe a different test case would be better.

How is Wayland vs X handled by Homebrew Core for Linux, though, if dependencies can't be optional?

Reply:
xclip is now non-optional for Linux

Reply:
Not quite sure how to handle Wayland vs X on Linux without optional Homebrew Core packages being allowed, though.